centreon/www/front_src/src/Dashboards/Dashboard/AddEditWidget/WidgetProperties/Inputs/Resources/useResources.ts

```diff
@@ -192,7 +195,7 @@ const useResources = (propertyName: string): UseResourcesState => {
     };
 
   useEffect(() => {
-    if (!singleResourceTypeSelection || !isEmpty(value)) {
+    if (!isEmpty(value)) {
```

Comment:
Does this work correctly with the single metric widget?

Reply:
yes sure!